### ISSUE TITLE
Production build error GeolocationEnabledComponent

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#136]
Production build error with `ionic cordova run browser --prod --release`:
```
ERROR in : Cannot determine the module for class GeolocationEnabledComponent in /Users/someuser/gitlab/aq-mobile-be/src/app/components/geolocation-enabled/geolocation-enabled.component.ts! Add GeolocationEnabledComponent to the NgModule to fix it.

[ERROR] An error occurred while running subprocess ng.

        ng run app:ionic-cordova-build:production --platform=browser exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

**Comment by opeeters** [comment]
same issue when doing `ionic cordova build android --prod --release`

**Comment by janschulte** [comment]
Fixed with: 07ec9fc8e8e27734e4080ed6694e847190e82da1
